packages/fe-fpm-writer/templates/building-block/rich-text-editor/View.xml

```diff
@@ -0,0 +1,14 @@
+<<%- macrosNamespace %>:RichTextEditorWithMetadata
+    metaPath="<%- data.metaPath %>"
+    id="<%- data.id %>">
+    <<%- macrosNamespace %>:buttonGroups>
+        <richtexteditor:ButtonGroup
+            name="<%- data.buttonGroup.name %>"
+            buttons="<%- data.buttonGroup.buttons %>"
+            visible="<%- data.buttonGroup.visible %>"
+            <% if (typeof data.buttonGroup.priority !== 'undefined') { %>priority="<%- data.buttonGroup.priority %>" <% } %>
+            <% if (typeof data.buttonGroup.customToolbarPriority !== 'undefined') { %>customToolbarPriority="<%- data.buttonGroup.customToolbarPriority %>" <% } %>
+            <% if (typeof data.buttonGroup.row !== 'undefined') { %>row="<%- data.buttonGroup.row %>" <% } %>
```

Comment:
@kjose90 
Could we please avoid whitespaces?
See example in storybook code snippet, if `priority`|`customToolbarPriority`|`row` is not defined, then we see new line(s):
![Uploading image.png…]()